lib/mix/tasks/dogma.ex

```diff
@@ -11,8 +11,11 @@ defmodule Mix.Tasks.Dogma do
     {dir, formatter} = argv |> parse_args
     config = Config.build
 
+    {:ok, events} = GenEvent.start_link([])
```

Comment:
I think if the testing mechanism fails in some fashion on CI then the build should fail.